docs/datacube/README.md

```diff
@@ -23,7 +23,7 @@ export default {
     data() {
         return {
             chart: new DataCube({
-                chart: [], onchart: [], offchart: []
+                chart: {}, onchart: [], offchart: []
```

Comment:
Haven't worked with DC yet, but am assuming it still expects `chart` as an array.